features/schedule/basic/feature.go

```diff
@@ -55,15 +55,20 @@ func Execute(ctx context.Context, r *harness.Runner) (client.WorkflowRun, error)
 	r.Require.Equal(workflowID, desc.Schedule.Action.(client.ScheduleWorkflowAction).ID)
 
 	// Confirm simple list
-	iter, err := r.Client.ScheduleClient().List(ctx, client.ScheduleListOptions{})
-	r.Require.NoError(err)
-	foundSchedule := false
-	for iter.HasNext() && !foundSchedule {
-		e, err := iter.Next()
+	// TODO: This should not require a retry loop. See https://github.com/temporalio/sdk-features/issues/182
+	listingErr := harness.RetryFor(10, 1*time.Second, func() (bool, error) {
```

Comment:
Alternatively, `r.Require.Eventually(...`, see https://pkg.go.dev/github.com/stretchr/testify/require#Eventually

Reply:
Nice to know